cairo/crates/contracts/tests/isms/test_aggregation.cairo

```diff
@@ -104,3 +114,65 @@ fn test_aggregation_verify() {
     assert(aggregation.verify(concat_metadata, message), 'Aggregation: verify failed');
 }
 
+
+#[test]
+fn test_aggregation_verify_e2e() {
+    let aggregation_threshold = 1;
+
+    // MESSAGEID
+    let message_body = BytesTrait::new(11, array![0x68656C6C6F20776F726C640000000000]);
```

Comment:
Where do these constants come from

Reply:
removed this test as it's duplicate of test_aggregation_verify

(test_aggregation_verify_e2e was meant to unit test the metadata generated in the katana setup.)